### DISCUSSION TITLE
[A-L] (2023/24) Foglio 11 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#220] [Esercizi - Bertini]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/7e320714-9ae4-48fb-8f56-0c7a4b18cc94)

**Comment by IonutCicio** [comment]
#### 1)

Sappiamo che la catena è ergodica, quindi la probabilità stazionaria $\pi$ _(con autovalore 1)_ è unica. Per trovare l'autovettore con autovalore 1 basta risolvere la seguente equazione

$$
\begin{pmatrix}
\pi(1) & \pi(2)
\end{pmatrix}
\cdot
\begin{pmatrix}
\frac{2}{3} & \frac{1}{3}\\
\frac{2}{5} & \frac{3}{5}
\end{pmatrix} =
\begin{pmatrix}
\pi(1) & \pi(2)
\end{pmatrix}
$$

Per cui si ha

$$
\begin{pmatrix}
\pi(1) & \pi(2)
\end{pmatrix}
\cdot
\begin{pmatrix}
\frac{2}{3} & \frac{1}{3}\\
\frac{2}{5} & \frac{3}{5}
\end{pmatrix} =
\begin{pmatrix}
\frac{2}{3}\pi(1)+\frac{2}{5}\pi(2) & \frac{1}{3}\pi(1) + \frac{3}{5}\pi(2)
\end{pmatrix}
$$

E bisogna risolvere il sistema

$$
\begin{cases}
\pi(1) = \frac{2}{3}\pi(1)+\frac{2}{5}\pi(2)\\
\pi(2) = \frac{1}{3}\pi(1) + \frac{3}{5}\pi(2)\\
\pi(1) + \pi(2) = 1
\end{cases}
\implies
\begin{cases}
\pi(1) = \frac{6}{11}\\
\pi(2) = \frac{5}{11}
\end{cases}
\implies
\pi =
\begin{pmatrix}
\frac{6}{11} & \frac{5}{11}
\end{pmatrix}
$$

#### 2)

$$
\lim\limits_{n\to\infty}  P^n =
\begin{pmatrix}
\frac{6}{11} & \frac{5}{11}\\
\frac{6}{11} & \frac{5}{11}
\end{pmatrix}
$$

> [!NOTE]
> TODO: lo devo dimostrare, ma basta vedere cosa succede nel caso generico per il vettore (0 1) e il vettore (1 0)
#### 3)

Sappiamo che la catena è ergodica, quindi, per il teorema ergodico sappiamo che 

$$
\lim\limits_{n\to\infty} 
\begin{pmatrix}
\frac{1}{2} & \frac{1}{2}
\end{pmatrix} P^n = \pi =
\begin{pmatrix}
\frac{6}{11} & \frac{5}{11}
\end{pmatrix}
$$
